template/src/core/network/fetcherUtils.ts

```diff
@@ -0,0 +1,5 @@
+function fetchJSONMiddleware(response: globalThis.Response) {
+  return response.json().catch((_error) => ({}));

```

Comment:
```suggestion
  return response.json().catch((error) => {
        console.error("Response may not support `json` conversion, so suppress the error");
        console.error(error);
        return {};
  });
```
I think we can at least log it to the console when this happens